frontendActions/build.gradle

```diff
@@ -16,3 +16,9 @@ dependencies {
 
 addIntellijToCompileClasspath(withGit4Idea: true)
 applySubtypingChecker()
+
+apply plugin: 'kotlin'
+compileKotlin {
+  destinationDir = compileJava.destinationDir
+  kotlinOptions.jvmTarget = "1.8"
```

Comment:
Hmm Java 8 and not Java 11? :thinking:

Reply:
If Java 8 is needed indeed, pls add a comment why this discrepancy

Reply:
Without setting this:
![image](https://user-images.githubusercontent.com/19799111/87657941-cb996300-c75b-11ea-9655-275689a08caf.png)

But it turned out that both `1.8` and `11` work fine so I am leaving the later (to avoid ambiguity)